codes/ruby/chapter_sorting/bubble_sort.rb

```diff
@@ -0,0 +1,50 @@
+=begin
+File: bubble_sort.rb
+Created Time: 2024-04-02
+Author: Cy (9738314@gmail.com)
+=end
+
+### 冒泡排序 ###
+def bubble_sort(nums)
+  n = nums.length()
+  # 外循环：未排序区间为 [0, i]
+  for i in (n - 1...0).step(-1)
```

Comment:
For better readability

```suggestion
  for i in ((n - 1)...0).step(-1)
```

But I think it is better if you go with this

```suggestion
  for i in (n - 1).downto(1)
```